AzureMonitorAgent/ama_tst/modules/error_codes.py

```diff
@@ -0,0 +1,30 @@
+# # General Errors
+NO_ERROR = 0
+USER_EXIT = 1
+# ERR_SUDO_PERMS = 100
+ERR_FOUND = 101
+# ERR_PYTHON_PKG = 154
+
+# Warnings
```

Comment:
I'd recommend only keeping in the warnings that you're currently using in the TST - you can treat warnings like errors that won't immediately block the rest of the tests from completing